spec/std/float_spec.cr

```diff
@@ -82,6 +82,17 @@ describe "Float" do
     assert { 1.0.fdiv(0).should eq 1.0/0.0 }
   end
 
+  describe "divmod" do
+    assert { 1.2.divmod(0.3)[0].should eq(4) }
+    assert { 1.2.divmod(0.3)[1].should be_close(0.0, 0.00001) }
+
+    assert { 1.3.divmod(0.3)[0].should eq(4) }
+    assert { 1.3.divmod(0.3)[1].should be_close(0.1, 0.00001) }
+
+    assert { 1.4.divmod(0.3)[0].should eq(4) }
+    assert { 1.4.divmod(0.3)[1].should be_close(0.2, 0.00001) }
+  end
```

Comment:
Ahh, good point—I did not have that in mind. I’ll add some more test cases and fix the implementation.